src/pages/e-ticket.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+// using https://github.com/dephraiim/qrcode
+// example: https://qr-service.vercel.app/api/code?text=https://freshmen2023.sgcu.in.th/&size=200
+
+const ETicket = () => {
+    return (
+        <div className="min-h-screen">
+            <div>
+                <Link href="https://freshmen2023.sgcu.in.th/profile">
+                    <button className="justify-left relative h-10 w-10 rounded-lg px-16 py-16">
+                        <div className="">
```

Comment:
This `div` is not necessary.

Reply:
What I meant is you don't need to place the image within this div. Doing so only overcomplicates the code.